### PR TITLE
[training] Nové školení "Začněte debugovat jako profíci s xdebugem!”

### PR DESCRIPTION
**Pull request by Lexinek**

### CODE REVIEW

File: source/_data/trainings.neon [hunk 1]
```diff
@@ -95,3 +95,22 @@ parameters:
...
+                <li>Webové aplikace</li>
+                <li>Latte a Twig šablony</li>
+                <li>Testy</li>
+                <li>CLI skripty</li>
+                <li>REST api požadavky</li>
+            </ul>
+            <p><em>Je velmi pravděpodobné, že po absolvování školení, si budete ťukat na čelo, jak jste doposud mohli bez xdebugu v PHP programovat :-).</em></p>
+            '''
+            duration: "6 hodin"
+            price: "3499 Kč"
```
Comment (TomasVotruba) [inline source/_data/trainings.neon]: 6 hodin je docela dost. Dal bych tu cenu 3999 Kč, to je taková klasika za takový čas

Comment (TomasVotruba) [reply source/_data/trainings.neon]: Jinak ready to merge :)

**Comment by TomasVotruba** [comment]
Thanks a lot for fast response, merged :+1:

**Comment by Lexinek** [comment]
I thank you for your feedback, i have always had problems with appropriate pricing 😄

**Comment by Lexinek** [comment]
anyways @TomasVotruba editation of the [google form](https://docs.google.com/forms/d/e/1FAIpQLSf-x6HlwpfIB-Ig_oyrPzk7PKbyF4YMIHLH21n_dV9V5Edmyw/viewform) is required 😉

**Comment by TomasVotruba** [comment]
Done! 

https://docs.google.com/forms/d/1axnUDWMpVU-6TnWqkmkupPxQRpLIhSAhxNay71yv1M8/edit

**Pull request merged**